### ISSUE TITLE
Training BiSeNet 

### ISSUE THREAD

**Issue by ashkanaev** [GeorgeSeif/Semantic-Segmentation-Suite#108]
Hello @GeorgeSeif!

Thanks for your work. I'm have a question about BiSeNet.  Tried to train with default settings next configuration: 
Dataset: CamVid
Arch: resnet50 + BiSenet
Image size: 512x512
The rest setting is default.

Max mAp achieved is ~85%. Examples of validation images:
![0001tp_006870_gt](https://user-images.githubusercontent.com/16134740/46218571-a4d32800-c34d-11e8-9e4a-a9423bbd5904.png)
![0001tp_006870_pred](https://user-images.githubusercontent.com/16134740/46218574-a69ceb80-c34d-11e8-968e-39bc70c12d75.png)

Have you test proposed BiSeNet on CamVid? Which configuration? What max mAP achieved?

Hope for your reply!
Thanks!

**Comment by Ji-qing** [comment]
Hi, I am a newer on semantic segmentation.  Could you tell me the specific details of training BiSeNet？
Thank you very much!

**Comment by shanmugaraj1986** [comment]
> I have tested it, albeit briefly (few epochs)
> 
> In terms of mAP, 85% is actually very high and the results you are getting on a 32 class segmentation task are considered "good", as 32 classes is quite challenging. The reason that your output looks a bit "messy" is probably a low IoU score. This usually occurs when you have some smaller objects, many classes, or in general your scene is quite challenging! Again overall the results you are getting would be considered quite good for the current setup on CamVid
> 
> Now there is a few things you can do to try and improve the results:
> 
> (1) Specialized training for higher IoU. This would involve something like a new loss function to try and give weight to the IoU. This option is **hard**, but can potentially work
> 
> (2) Make your image sizes bigger. The CamVid image sizes are quite big. When you crop to 512x512, you don't really get the full picture. Generally speaking, working with higher resolution images usually helps with getting better results. If your GPU can handle it, try increasing the crop size!
> 
> (3) Reduce the number of classes. Having less classes always makes the problem easier. Perhaps there are some classes that you don't care about seeing, such as "animal". You can just convert this class to the background class then! You might have 2 classes which are similar and that you don't care to differentiate between, such as road vs road shoulder, so then you can just fuse those two together
> 
> Try those out and see if your results improve! Since this is not a bug or feature request, I will close this issue but continue replying to your comments :)

Hi Bro, Where do you get the BiSeNet code, Please share the path. Thanks
